### ISSUE TITLE
checking out git as svn

### ISSUE THREAD

**Issue by gforney** [gforney/FDS-SMVgit#2]
fyi:  I checked out the git repository using
svn co https://github.com/gforney/FDS-SMVgit/trunk FDS-SMVgit2

(as kevin guessed, I added 'trunk' to the end of the url they provided  ( https://github.com/gforney/FDS-SMVgit ) and specified the directory (FDS-SMVgit2 ) where I wanted to put it

randy, kevin and rick did you get this as an email?

**Comment by gforney** [comment]
another problem (that rick has identified) is with line endings.  files have line endings appropriate for linux of the mac not the PC .  we need to figure out how to get git to switch line endings depending on what platform the repository is on

**Comment by mcgratta** [comment]
I got it as an email.

**Comment by CFAST-NIST** [comment]
yep, as an email

On May 12, 2015, at 10:38 AM, gforney <notifications@github.com<mailto:notifications@github.com>> wrote:


fyi: I checked out the git repository using
svn co https://github.com/gforney/FDS-SMVgit/trunk FDS-SMVgit2

(as kevin guessed, I added 'trunk' to the end of the url they provided ( https://github.com/gforney/FDS-SMVgit ) and specified the directory (FDS-SMVgit2 ) where I wanted to put it

randy, kevin and rick did you get this as an email?

**Comment by CFAST-NIST** [comment]
The line endings work fine if you use smartgit rather than smartsvn .. I've been trying a number of different attempts to get SVN to recognize the line endings ... still working on it.

**Comment by gforney** [comment]
I've got a solution for the source code<--> executable mapping problem -
along the lines of what we talked about.

I've written a batch file that gets the revision number of a given
directory and returns it into an environment variable.  I then will write
another batch file that will create an appropriate fortran .f90 file
containing this revision number in some character string accessible to the
rest of the program (haven't written this yet).

The make_fds.bat (or make_cfast.bat) script now will pass the name of this
created fortran file to the Makefile.  The neat thing is in the Makefile
the default name of this file is blank.  So, if the file is not created the
program will build fine.

I was doing this for FDS but think I'll do it for cfast instead.

On Tue, May 12, 2015 at 12:51 PM, CFAST-NIST <notifications@github.com>
wrote:

> The line endings work fine if you use smartgit rather than smartsvn ..
> I've been trying a number of different attempts to get SVN to recognize the
> line endings ... still working on it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gforney/FDS-SMVgit/issues/2#issuecomment-101346007>.
>



-- 
Glenn Forney
